src/briefcase/integrations/xcode.py

```diff
@@ -469,12 +470,13 @@ def get_identities(command, policy):
             ['security', 'find-identity', '-v', '-p', policy],
         )
 
-        identities = dict(
-            IDENTITY_RE.match(line).groups()
-            for line in output.split('\n')
-            if IDENTITY_RE.match(line)
-        )
+        return {
+            (
+                IDENTITY_RE.match(line).groups()
+                for line in output.split('\n')
+                if IDENTITY_RE.match(line)
+            )
+        }
```

Comment:
The brackets are a problem here; this will be returning a set of pairs, not a dict. (and I think this is the source of the smoke test failures)

Reply:
Nice catch!